build.fsx

```diff
@@ -216,6 +217,30 @@ Target "Run" (fun _ ->
 )
 
 
+Target "RunSSR" (fun _ ->
+    runDotnet clientPath "restore"
```

Comment:
why do we need this? what is it doing?

Reply:
This will set a compile directive `DEBUG_SSR` to use webpack-dev-server's bundle file path in the server-rendered path. Ideally, in `RunSSR` we can trigger client and server to rebuild when we edit any file, so we can dev and debug with SSR; but in normal `Run` we would only trigger the server to rebuild when we edit back-end files, and we can dev with webpack-dev-server and use the server code as an API service.

But I cannot pass the `configuration` to dotnet-watch to make it work in different command, currently, we totally ignored client files' change when `watch run` the server.

https://github.com/SAFE-Stack/SAFE-BookStore/pull/306/files#diff-694c6c636cd70bcc23aa716434744995R12
https://github.com/SAFE-Stack/SAFE-BookStore/pull/306/files#diff-3b3c3fd8d4441c71716037ce14cb42b0R14